### ISSUE TITLE
Audit site to identify accessibility improvements

### ISSUE THREAD

**Issue by katjam** [geeksforsocialchange/gfsc-v3#211]
## Description
Run down accessibility checklist to make sure we're doing all the right things. Obviously we will be accessibility first as we build but it's worthwhile to do an audit after we have built a few pages.

### Must do: 
- [x] #210 
- [x] Use a WAVE tool to evaluate each page https://wave.webaim.org/
- [x] Run through site use with keyboard only
- [ ] #326
- [x] Check all meaningful images for alt text - add blank alt for non-menaingful
- [x] Check all form inputs for labels
- [x] Check all links for meaningful & unique link text
- [x] Check all headings are correct for thier semantic meaning

### Should do:
- [ ] Run down https://www.w3.org/WAI/WCAG21/quickref and ticket anything that does not meet AA

## Notes
- Starling bank has a nice statement https://www.starlingbank.com/website-accessibility. We ought to be doing all the things listed there as "What we are doing"

---

#### all pages
- [ ] #306
![image](https://github.com/geeksforsocialchange/gfsc-v3/assets/6824891/bc06e900-e37e-47c1-8aa8-a95db2a5c1be)

#### homepage
- [ ] #308
- [ ] #307

#### studio
- [ ] #309
- [ ] #308

#### our work
- [ ] #308
- [x] empty link and UL generated by themes under Trans Dimension, possibly other projects that have no themes
- [x] #311
![image](https://github.com/geeksforsocialchange/gfsc-v3/assets/6824891/c74c0c96-f45e-4ba8-a43e-a7a4ae5c3a29)

#### team
- [ ] #308

#### support
- [ ] #308

#### blog
- [ ] #308
- [x] #311

#### blog - Everything we want to tell you about your funding scheme...
- [ ] #313
- [ ] #312
![image](https://github.com/geeksforsocialchange/gfsc-v3/assets/6824891/de326dac-7b67-4c32-9539-d97124c1c9bf)

#### blog - tech [...] magic wand
- [ ] #315
![image](https://github.com/geeksforsocialchange/gfsc-v3/assets/6824891/79b74a0e-c9e6-42a1-bca6-f9973ca9e854)
- [ ] #314
![image](https://github.com/geeksforsocialchange/gfsc-v3/assets/6824891/268e6c62-f7f0-4803-bd12-b0ce522ae280)

#### blog - The Trans Dimension Guide to Inclusive Events
- [ ] #316
- [ ] #317

#### project - pube
- [ ] #318
- [ ] #308
- [ ] #319

#### project - pube
- [ ] #310
![image](https://github.com/geeksforsocialchange/gfsc-v3/assets/6824891/ac51d45d-833b-413c-9f3b-1f981af3242f)
- [ ] #321
![image](https://github.com/geeksforsocialchange/gfsc-v3/assets/6824891/c8148e87-54af-464a-b835-118e40921943)

#### contact
- [ ] #308

#### homepage
- [ ] #320

#### our work
- [ ] #322

https://github.com/geeksforsocialchange/gfsc-v3/assets/6824891/2ec835fe-5608-4c87-8b16-0031f889f9b2

- [ ] #323

https://github.com/geeksforsocialchange/gfsc-v3/assets/6824891/e56b668d-8d2e-4a59-9ce1-cbcce2278cad

### startling list
- [ ] #324
> On every page, users can bypass the navigation menu and skip straight to the main content, if they want to.
- [ ] #325
